### PR TITLE
Implement feedback controller

### PR DESCRIPTION
**Pull request by Nikityanov**
Implement feedback service and controller.

### CODE REVIEW

File: InternshipBE/BL/DTOs/FeedbackDTO.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        public int CandidateId { get; set; }
+
+        public EnglishLevelType EnglishLevelType { get; set; }
+
+        public DateTime Date { get; set; }
+
+        public string Description { get; set; }
+
+        public ICollection<Evaluation> Evaluations { get; set; }
```
Comment (romanstat) [inline InternshipBE/BL/DTOs/FeedbackDTO.cs]: ?

Comment (Nikityanov) [reply InternshipBE/BL/DTOs/FeedbackDTO.cs]: I just think about it and it's implementation. So i copy it from Entity model.

File: InternshipBE/DAL/Repositories/EFUnitOfWork.cs [hunk 2]
```diff
@@ -33,6 +34,16 @@ public IInternshipRepository Internships
...
+		public IFeedbackRepository Feedbacks
+        {
+            get
+            {
+				if(_feedbackRepository == null)
+					_feedbackRepository = new FeedbackRepository(_db);
+				return _feedbackRepository;
+            }
+        }
+
```
Comment (romanstat) [inline InternshipBE/DAL/Repositories/EFUnitOfWork.cs:46]: image
Alignment?

File: InternshipBE/WebApi/Controllers/FeedbackController.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FeedbackController : Controller
+    {
+        private readonly IFeedbackService _feedbackService;
+        public FeedbackController(IFeedbackService feedbackService)
+        {
+            _feedbackService = feedbackService;
+        }
+        [HttpGet("GetFeedbackById")]
```
Comment (romanstat) [inline InternshipBE/WebApi/Controllers/FeedbackController.cs]: space

File: InternshipBE/WebApi/Controllers/FeedbackController.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        public async Task<IActionResult> CreateFeedback ([FromBody] FeedbackDTO feedback)
+        {
+            return Ok(await _feedbackService.CreateFeedbackAsync(feedback));
+        }
+
+        [HttpPost("UpdateFeedback")]
+        public async Task<IActionResult> UpdateFeedback ([FromBody] FeedbackDTO feedback)
+        {
+            return Ok(await _feedbackService.UpdateFeedbackAsync(feedback));
+        }
```
Comment (romanstat) [inline InternshipBE/WebApi/Controllers/FeedbackController.cs]: camel case endpoint names

File: InternshipBE/WebApi/Controllers/FeedbackController.cs [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+        [HttpPost("CreateFeedback")]
+        public async Task<IActionResult> CreateFeedback ([FromBody] FeedbackDTO feedback)
+        {
+            return Ok(await _feedbackService.CreateFeedbackAsync(feedback));
+        }
+
+        [HttpPost("UpdateFeedback")]
+        public async Task<IActionResult> UpdateFeedback ([FromBody] FeedbackDTO feedback)
```
Comment (romanstat) [inline InternshipBE/WebApi/Controllers/FeedbackController.cs]: image

**Review by romanstat — APPROVED** [review]

**Review by ihhyy — APPROVED** [review]

**Review by vldmrklnk — APPROVED** [review]

**Pull request merged**